tests/test_bot.py

```diff
@@ -0,0 +1,49 @@
+import time
+import pytest
+
+
+TIMEOUT = 60
+
+
+class TestAdminGroup:
+    def create_admingroup(self, admbot, botuser, db):
+        admchat = admbot.create_group_chat("admins", [], verified=True)
+        with db.write_transaction() as conn:
+            conn.set_config("admingrpid", admchat.id)
+        qr = admchat.get_join_qr()
+        chat = botuser.qr_join_chat(qr)
+        while not chat.is_protected():
+            time.sleep(0.1)
```

Comment:
If you activated the FFIEventTracker then you can now do:
```
    botuser._evtracker.wait_securejoin_inviter_progress(1000)
```

Reply:
good idea, but I now think it makes more sense to wait here until 5 messages have arrived. This makes the tests after the fixture a lot less flaky, as the welcome message can't slide into the test anymore.